src/package/nuspec/TestPlatform.TestHost.nuspec

```diff
@@ -93,6 +98,24 @@
 
     <file src="Microsoft.TestPlatform.TestHost\netcoreapp2.1\Microsoft.TestPlatform.TestHost.props" target="build\netcoreapp2.1\" />
     
+    <!-- net6.0 -->
+    <file src="Microsoft.TestPlatform.TestHost\net6.0\Microsoft.VisualStudio.TestPlatform.*.dll" target="lib\net6.0\" />
+    <file src="Microsoft.TestPlatform.TestHost\net6.0\Microsoft.TestPlatform.*.dll" target="lib\net6.0\" />
+    <file src="Microsoft.TestPlatform.TestHost\net6.0\testhost.dll" target="lib\net6.0\" />
+    <file src="Microsoft.TestPlatform.TestHost\net6.0\testhost.deps.json" target="lib\net6.0\" />
+    <file src="Microsoft.TestPlatform.TestHost\net6.0\x86\msdia140.dll" target="lib\net6.0\x86\" />
+    <file src="Microsoft.TestPlatform.TestHost\net6.0\x64\msdia140.dll" target="lib\net6.0\x64\" />
+
+    <file src="Microsoft.TestPlatform.TestHost\net6.0\Microsoft.TestPlatform.PlatformAbstractions.dll" target="build\net6.0\" />
+    <file src="Microsoft.TestPlatform.TestHost\net6.0\win7-x64\testhost.dll" target="build\net6.0\x64\" />
+    <file src="Microsoft.TestPlatform.TestHost\net6.0\win7-x64\testhost.exe" target="build\net6.0\x64\" />
+    <file src="Microsoft.TestPlatform.TestHost\net6.0\win7-x64\Microsoft.TestPlatform.PlatformAbstractions.dll" target="build\net6.0\x64\" />
+    <file src="Microsoft.TestPlatform.TestHost\net6.0\win7-x86\testhost.x86.dll" target="build\net6.0\x86\" />
```

Comment:
The `win7-*` runtime switch seems weird for .NET Core. I see that's what we have for netcoreapp2.1 too but I'm trying to understand how this works in a cross-plat runtime. Any thoughts?

Reply:
It was added here: https://github.com/microsoft/vstest/pull/217/files and it looks like it's coming from a "template" to migrate to csproj. Imho this does not affect how our libs are built, it would just package them with appropriate runtime, which we then ignore because we copy only our dlls (which are still anycpu) to the result folder -> nuget package. So imho this is totally okay, even if not the cleanest way probably.